src/component_macro_reference.md

```diff
@@ -214,6 +214,6 @@ let widgets = view_output!();
 
 ### Manual view
 
-You can also implement your own view logic that's added to the view code the view update code generated by the macro macro generates. Code inside `pre_view()` will run before the view update, while `post_view()` will run after.
+You can also implement your own view logic, which will be added to the view code that the view update code generates. Code inside `pre_view()` will run before the view update, while `post_view()` will run after.
```

Comment:
which will be added to the view code that the **view macro** generates.

Reply:
changed